scenes/player/player.gd

```diff
@@ -50,11 +60,9 @@ func _physics_process(delta):
 	if FUEL > 0:
 		velocity -= velocity_update
 		FUEL -= abs(velocity_update.x) + abs(velocity_update.y)
-		if $AudioStreamPlayer2D.pitch_scale == 0.7:
-			$AudioStreamPlayer2D.pitch_scale = 1
 	else:
-		$AudioStreamPlayer2D.pitch_scale = 0.7
-		
+		$AudioStreamPlayer2D.pitch_scale = background_music_pitch_scale_slow
```

Comment:
Emit a signal somewhere when the player dies, and the rest of the scenes can use that.